bin/jspmjasmineinspect.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node --inspect
```

Comment:
The `--inspect` is explained [here](https://mattdesl.svbtle.com/debugging-nodejs-in-chrome-devtools)